doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,618 @@
+    * nRF91 fails to receive large packets (over 4000 bytes) over NB-IoT. LTE-M is not affected.
+    * nrf_connect fails if called immediately after initialization of the device. A delay of 1000 ms is required for this to work as intended.
+
+Crypto
+======
+
+.. note::
+   Old - still valid?
+
+   * The :ref:`nrfxlib:nrf_security` glue layer is broken because symbol renaming is not handled correctly.
+     Therefore, the behavior is undefined when selecting multiple back-ends for the same algorithm (for example, AES).
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* :option:`CONFIG_BT_HCI_TX_STACK_SIZE` must be set to 1536 when selecting :option:`CONFIG_BT_LL_NRFXLIB`.
+* The :ref:`nrfxlib:ble_controller` 0.3.0-3.prealpha might assert when receiving a packet with an CRC error on LE Coded PHY after performing a DLE procedure where RX Octets is changed to a value above 140.
+* :option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE` must be set to 2048 when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`central_uart` and :ref:`central_bas`.
+* :option:`CONFIG_NFCT_IRQ_PRIORITY` must be set to 5 or less when selecting :option:`CONFIG_BT_LL_NRFXLIB` on :ref:`peripheral_hids_keyboard`.
+* When selecting :option:`CONFIG_BT_LL_NRFXLIB`:
+  If a directed high duty cycle advertiser times out, the application might have to wait a short time before starting a new connectable advertiser.
+  Otherwise, starting the advertiser will fail.
+* Bluetooth Low Energy peripheral samples are unstable in some conditions (when pairing and bonding are performed and then disconnections/re-connections happen).
+* When running the :ref:`bluetooth_central_dfu_smp` sample, the :option:`CONFIG_BT_SMP` configuration must be aligned between this sample and the Zephyr counterpart (:ref:`zephyr:smp_svr_sample`). However, security is not enabled by default in the Zephyr sample.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+* On some operating systems, the nrf_desktop application is unable to reconnect to a host.
+* central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
+* On nRF51 devices, BLE samples that use GPIO might crash when buttons are pressed frequently. In such case, the GPIO ISR introduces latency that violates real-time requirements of the Radio ISR. nRF51 is more sensitive to this issue than nRF52 (faster core).
+
+
+Bootloader
+----------
+
+* Public keys are not revoked when subsequent keys are used.
+* The bootloader does not work properly on nRF51.
+
+.. note::
+   Old - which of these are still valid?
+
+    * Building and programming the immutable bootloader (see :ref:`ug_bootloader`) is not supported in SEGGER Embedded Studio.
```

Comment:
@hakonfam @andvib Please advise on this and the next.

Reply:
I did some quick testing just now, and it seems they're both kinda valid.

From before I had only tested with SES in Windows for nrf9160_pca10090ns, which works. For nrf9160_pca10090 and nrf52840_pca10056 however it did not work. The immutable bootloader works for all the three boards when I compiled it with GCC in Linux.

So to me it seems like the safest and clearest will be to keep both of these points.